### DISCUSSION TITLE
Unable to download via FTP

### DISCUSSION THREAD

**Discussion by thistlillo** [iterate-ch/cyberduck#16502] [Q&A]
I need to access and download files from an FTP server, that requires authentication. I cannot share either the address or other information.

I have never been able to download files from that server via Cyberduck (I have been trying for years). Every time, I need to switch to FileZilla, that has no problems with the same server and files.

This is what has been happening with Cyberduck for some years:

- I can always connect to the FTP server (port 21, TLS) and browse its folder tree.
- When I select and ask to download a file, the download starts, but the connection gets interrupted every once in a while for nearly all the download attempts. 
- The download never stops, not even when there were no network issues (interruptions). I end up downloading 101% (and more) of a file. 

Out of comparison: with FileZilla, the connection never interrupts and the file is  downloaded, with the download stopping at 100%.

I am aware this is one of the worst bug reports ever filed. So, please, let me know if I can provide details (log, ports, etc) in case you decide to look into it.

Working on a Mac with Sonoma 14.2.1 (23C71) and the latest version of Cyberduck (but this report applies to previous versions of the OS and of FileZilla as well).

**Comment by crowell-tailord** [comment]
Same here, in some respects. I didn't follow my process as closely, but I'll go to download a ~500mb csv file, leave it for a second then come back and see its at like 120% downloaded. And I could let it run forever. It never stops, nor do I ever receive the actual file in my Finder.

**Comment by dkocher** [comment]
Try disabling [segmented downloads](https://docs.cyberduck.io/cyberduck/download/#segmented-downloads) in _ Preferences → Transfers → General → Downloads → Segmented download with multiple connections for file_.
